components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/domain.jsp

```diff
@@ -26,7 +26,7 @@
         if (request.getParameter("authFailureMsg") != null) {
             errorMessage = request.getParameter("authFailureMsg");
 
-            if (errorMessage.equalsIgnoreCase("domain.unknown")) {
+            if ("domain.unknown".equalsIgnoreCase(errorMessage)) {
```

Comment:
better to void of using string middle of the code as much as you can , also IMO this error message should be support for localization isn't it ?